contrib/syslog-ng-debun

```diff
@@ -1679,6 +1721,8 @@ if [ -n "$privacy_mode" ]; then
 	acquire_syslog_nprv
 else
 	acquire_syslog_all
+	acquire_all_files
```

Comment:
This would be be better moved inside acquire_syslog_all function. That would leave the branching on `privacy_mode` clean.

Reply:
@furiel : ok